### DISCUSSION TITLE
Matek H743 Slim no current values in OSD  since update to INAV 4.0 Stable

### DISCUSSION THREAD

**Discussion by terryfritz99** [iNavFlight/inav#7697] [Q&A]
## Current Behavior
Matek H743 Slim flight controller isn't reporting a value for current in telemetry since update to INAV 4.0

## Steps to Reproduce

1.Update firmware from INAV 4.0 RC2 to INAV 4.0 Stable
2.
3.
4.

## Expected behavior
My setup had been reporting the value for current flow in RC1 and RC2 releases. I have two quads to the same specs and both show the same problem.

## Suggested solution(s)
 No suggestion

## Additional context

https://pastebin.com/cuX3MRKG

---
****
INAV/MATEKH743 4.0.0 Dec 15 2021 / 15:18:48 (c20a9852)
# GCC-10.2.1 20201103 (release)

- FC Board name and vendor: Matek H743 Slim
- INAV version string:INAV/MATEKH743 4.0.0 Dec 15 2021 / 15:18:48 (c20a9852)
# GCC-10.2.1 20201103 (release)

**Comment by DzikuVx** [comment]
OK, for a minute I thought you found a problem and for a better part of an hour I was testing on one of my quads. And at the end, it ended up with nothing as current meter works with 4.0 and OSD was showing everything like it should.

There were no changes between RC2 and final that could have caused this and to be honest, I thought it was introduced around 3.0.2
But it works

**Reply by terryfritz99** [reply]
OK thanks Pawel; I'll dig deeper.
